Extensions/TopDownMovementBehavior/Extension.cpp

```diff
@@ -358,7 +358,22 @@ void DeclareTopDownMovementBehaviorExtension(gd::PlatformExtension& extension) {
           gd::ParameterOptions::MakeNewOptions().SetDescription(
               _("Angle to compare to (in degrees)")))
       .MarkAsAdvanced()
+      .SetHidden()
```

Comment:
I think this should not be deprecated. It's a valid use case, but we should have a way in the editor to show that "well, you probably want the condition with the tolerance for 90% of use cases"

Reply:
There is still the expression to do calculus with the actual value and it can be used in a compare condition.